src/test/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowSnykScanListenerLSlTest.kt

```diff
@@ -0,0 +1,157 @@
+package io.snyk.plugin.ui.toolwindow
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import com.intellij.ui.treeStructure.Tree
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.unmockkAll
+import io.snyk.plugin.Severity
+import io.snyk.plugin.pluginSettings
+import io.snyk.plugin.resetSettings
+import io.snyk.plugin.ui.toolwindow.nodes.root.RootOssTreeNode
+import io.snyk.plugin.ui.toolwindow.nodes.root.RootQualityIssuesTreeNode
+import io.snyk.plugin.ui.toolwindow.nodes.root.RootSecurityIssuesTreeNode
+import junit.framework.TestCase
+import org.eclipse.lsp4j.Position
+import org.eclipse.lsp4j.Range
+import snyk.common.ProductType
+import snyk.common.lsp.CommitChangeLine
+import snyk.common.lsp.DataFlow
+import snyk.common.lsp.ExampleCommitFix
+import snyk.common.lsp.ScanIssue
+import javax.swing.JTree
+import javax.swing.tree.DefaultMutableTreeNode
+
+class SnykToolWindowSnykScanListenerLSlTest : BasePlatformTestCase() {
```

Comment:
These tests pass regularly locally and in the Mac runner but randomly fail in Ubuntu or Windows in the pipeline:
```
io.snyk.plugin.ui.toolwindow.SnykToolWindowSnykScanListenerLSlTest > testAddInfoTreeNodes does not add new tree nodes for code security if ignores are not enabled FAILED
    io.mockk.MockKException: Missing mocked calls inside every { ... } block: make sure the object inside the block is a mock
        at io.mockk.impl.recording.states.StubbingState.checkMissingCalls(StubbingState.kt:14)
        at io.mockk.impl.recording.states.StubbingState.recordingDone(StubbingState.kt:8)
        at io.mockk.impl.recording.CommonCallRecorder.done(CommonCallRecorder.kt:47)
        at io.mockk.impl.eval.RecordedBlockEvaluator.record(RecordedBlockEvaluator.kt:64)
        at io.mockk.impl.eval.EveryBlockEvaluator.every(EveryBlockEvaluator.kt:30)
        at io.mockk.MockKDsl.internalEvery(API.kt:94)
        at io.mockk.MockKKt.every(MockK.kt:1[43](https://github.com/snyk/snyk-intellij-plugin/actions/runs/9346896888/job/25723276168#step:6:44))
        at io.snyk.plugin.ui.toolwindow.SnykToolWindowSnykScanListenerLSlTest.mockScanIssues(SnykToolWindowSnykScanListenerLSlTest.kt:50)
        at io.snyk.plugin.ui.toolwindow.SnykToolWindowSnykScanListenerLSlTest.testAddInfoTreeNodes does not add new tree nodes for code security if ignores are not enabled(SnykToolWindowSnykScanListenerLSlTest.kt:126)
```


Seems to be related to https://github.com/mockk/mockk/issues/1015. I don't know if I should add an `@Ignore` for now or just remove the tests altogether